src/lib.rs

```diff
@@ -290,83 +290,87 @@ pub trait Miniconf {
     /// # Returns
     /// The number of bytes used in the `data` buffer for serialization.
     fn get(&self, path: &str, data: &mut [u8]) -> Result<usize, Error> {
-        self.string_get(path.split('/').peekable(), data)
+        self.get_path(path.split('/').peekable(), data)
     }
 
-    /// Create an iterator to read all possible settings paths.
+    /// Create an iterator of all possible paths.
+    ///
+    /// This is a depth-first walk.
     ///
     /// # Note
     /// The state vector can be used to resume iteration from a previous point in time. The data
     /// should be zero-initialized if starting iteration for the first time.
     ///
     /// # Template Arguments
-    /// * `TS` - The maximum number of bytes to encode a settings path into.
+    /// * `TS` - The maximum number of bytes to encode a path into.
```

Comment:
I think the only thing this PR is missing is an entry in the CHANGELOG indicating that the API for `Miniconf` functions has changed to be agnostic to usage (e.g. now referring to namespace paths and values)

Reply:
Ack. And I wanted to look over the docstrings anyway as well.